libkineto/include/IActivityProfiler.h

```diff
@@ -91,6 +93,12 @@ class IActivityProfilerSession {
   // processes trace activities using logger
   virtual void processTrace(ActivityLogger& logger) = 0;
 
+  virtual void processTrace(ActivityLogger& logger,
```

Comment:
is your intent here to have a default implementation that ignores the getLinkedActivity() callback?

Reply:
yea, i replace the function in caller with this new one. Add a defalut implementation in order to not break others child ProfilerSession.